src/other/Table.js

```diff
@@ -861,7 +862,18 @@
             this._selectionBag.click(selObj, { ctrlKey: this._selectionBag.isSelected(selObj) });
             this._selectionPrevClick = d;
         }
-        this.render();
+        if(this.updateOnRowSelection()){
+            this.render();
+        } else {
+            var context = this;
+            this.tbody.selectAll("tr.tr_" + this.id())
+            .classed("selected", function (_d) {
+                var d = _d.row;
+                return context._selectionBag.isSelected(context._createSelectionObject(d));
+            })
+            .classed("trId" + this._id, true)
```

Comment:
Looks good - Indentation is a bit off though (you might want to enable `editor.formatOnSave` in your vscode settings?

Reply:
@GordonSmith I've updated it with better indentation. I have the format on save feature disabled for my `HPCCJS_1_20_x` workspace, otherwise every PR would contain tons of changed lines due to formatting (I'll enable it if you're fine with that).

Reply:
Ahhh fair point...